src/analysis/flux_variability_analysis.jl

```diff
@@ -115,7 +115,7 @@ function flux_variability_analysis(
 
     flux_variability_analysis(
         model,
-        reaction_variables(model)[:, flux_indexes],
+        reaction_variables_matrix(model)[:, flux_indexes],
```

Comment:
at the price of making things a lot more verbose, how about `reaction_variables_mapping_matrix`?

Reply:
"mapping" already sounds like "bipartite"... I'll keep it short now